### ISSUE TITLE
make variable finder for debugging MOI models

### ISSUE THREAD

**Issue by claytonpbarrows** [NREL/PowerSimulations.jl#129]
This command gets the variable name after the solver returns an `infeasibility in R133`:
```
ED_copperplate.canonical_model.JuMPmodel.moi_backend.model_cache.model.var_to_name[JuMP.MOI.VariableIndex(113)]

"Pth_{323_CC_2,2}"
```

**Comment by sourabhdalvi** [comment]
First when it says, infeasible Row (Rxx) its referring to a constraint that's infeasible . Since we have all our constraints generated as anonymous constraints, the `model.con_to_name` is empty. I'm still not able to backtrack the infeasible row.

**Comment by jd-lara** [comment]
The best thing to do about it is to ask in the JuMP gitter

**Comment by jd-lara** [comment]
@sourabhdalvi an option is to add a `JuMP.set_name` for constraints for debugging. This probably should be optional since we don't want to overload the model building.

**Comment by jd-lara** [comment]
use this function. I will add a cleaner version later in the code base 
```Julia
function constraint_index(op_model::OperationModel)
    con_index = []
    for (k,v) in op_model.canonical_model.constraints
        for (idx,constraint) in enumerate(v)
            moi_index = JuMP.optimizer_index(constraint);
            push!(con_index,(k, idx, moi_index.value))
        end
    end
    return con_index
end
```

**Comment by jd-lara** [comment]
closes in favor of #160
